doc/Makefile.in

```diff
@@ -134,8 +135,11 @@ XML_SOURCES = \
 	reference_output.xml \
 	reference_processing.xml \
 	reference_raster.xml \
+	reference_relation.xml \
```

Comment:
in databases. "relation" means "table" :(

Reply:
Ha!  That's only internal, but still a good point.  I'll change it to "relationship".